README.md

```diff
@@ -172,6 +172,13 @@ Configuration is set using a json file stored in `config/config.json` of the bui
 
 An example `config.json` can be seen in the public folder in `config/config.example.json`.
```

Comment:
This files doesn't exist anymore. Can we create it? Also, the text above this doesn't quite make sense to me, so wonder if we can get it updated?

Reply:
Went looking for this because it's important and I didn't think it got deleted. It has moved to /public/config/config.example.json - which means this should be updated, yes

Reply:
actually, I think it's in the location it states, but it's somewhat poorly written - i.e.: look in the public folder, there you'll find `config.example.json` ... :-) I think we can get this updated to also include a link to the file